packages/shared-ux/file/file_picker/impl/src/file_picker_state.ts

```diff
@@ -32,25 +17,25 @@ export class FilePickerState {
   /**
    * Files the user has selected
    */
-  public readonly selectedFiles$ = new BehaviorSubject<FileJSON[]>([]);
+  public readonly selectedFiles$ = new Rx.BehaviorSubject<FileJSON[]>([]);
```

Comment:
I think the common practice is that we import `rxjs` modules, like it was done previously, so am not sure why this change?

Reply:
I was not entirely sure what the common practice is TBH, I've seen it used both ways. This way may net more chars (since each operator/observable/etc is namespaced), but it removed a lot of what I think of as "preamble noise". Opening the file now, you get straight to the important bits of content, which is something I think is worth adding the namespaced uses for.

This is a strong opinion, but weakly held. Happy to go with the previous approach if you prefer it, just wanted to share the reasoning :)